backends/aerospike.go

```diff
@@ -110,6 +85,45 @@ func NewAerospikeBackend(cfg config.Aerospike, metrics *metrics.Metrics) *Aerosp
 	}
 }
 
+// generateAerospikeClientPolicy returns an Aerospike ClientPolicy object configured according to values
+// in config.Aerospike fields
+func generateAerospikeClientPolicy(cfg config.Aerospike) *as.ClientPolicy {
+	clientPolicy := as.NewClientPolicy()
+	// cfg.User and cfg.Password are optional parameters
+	// if left blank in the config, they will default to the empty
+	// string and be ignored
+	clientPolicy.User = cfg.User
+	clientPolicy.Password = cfg.Password
+
+	// Aerospike's connection idle deadline default is 55 seconds. If greater than zero, this
+	// value will override
+	if cfg.ConnIdleTimeoutSecs > 0 {
+		clientPolicy.IdleTimeout = time.Duration(cfg.ConnIdleTimeoutSecs) * time.Second
+	}
+
+	// Aerospike's default connection queue size per node is 256.
+	// If cfg.ConnQueueSize is greater than zero, it will override the default.
+	if cfg.ConnQueueSize > 0 {
```

Comment:
I don't think this comment is necessary, the well-written code explains it I think.

Reply:
Agree. Probably a leftover. I sometimes structure logic into comments before writing the actual code. Removed.